commit 0d38971ed7ed7d6ad413545dcd6476bbaf6c36da (refs/changes/99/399/5)
Author: David Pacheco <dap@joyent.com>
Date:   2016-09-02T09:59:28-07:00 (3 years, 1 month ago)
    
    this is a test change
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/README.md b/README.md
index f6da383..75cf6c7 100644
--- a/README.md
+++ b/README.md
@@ -1,4 +1,3 @@
 # gerrit-test
 
 This is a dummy repository for verifying cr.joyent.us behavior.
-Test update.
